client/src/components/Checkout.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class Checkout extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      rooms: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/rooms/1/reservations') //hard coded TODO refactor to be dynamic
+      .then((response) => {
+        return response.json();
+      })
```

Comment:
@Pfdiamond13 @fradleybrancis using AirBnB style guide linting? Because it would say that the callback can be refactored to one line:
```
.then(response => response.json())
```
"response" should be more semantic, like "reservations"